changelog.html

```diff
@@ -59,6 +59,10 @@
   <li class=bug>
     <code>IllegalStateException</code> from Winstone when making certain requests with access logging enabled.
     (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37625">issue 37625</a>)
+  <li class=bug>
+    Failure to serialize a single <code>Action</code> could cause an entire REST export response to fail.
+    Upgraded to Stapler <code>1.249</code> with a fix.
```

Comment:
Well, 1.249 had the fix for the stated bug, 1.250 just corrected an obscure regression introduced by an unrelated change in 1.249.